packages/node/src/db/AccountRepository.test.ts

```diff
@@ -34,74 +34,79 @@ describe(AccountRepository.name, () => {
   ]
 
   beforeEach(async () => {
-    repository.deleteAll()
+    await database.migrate()
```

Comment:
Why do we migrate here?

Reply:
I wanted to migrate in the db setup function for tests, but that function is synchronous and it would be a larger change to make it async so I opted for the lesser of two evils. Ideally we only migrate once in a separate setup step. I think we have something like this in the l2beat repo.